heron/common/src/python/utils/misc/communicator.py

```diff
@@ -12,12 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 '''communicator.py: module responsible for communication between Python heron modules'''
+import sys
```

Comment:
A bunch of the files seems to be instance related. Why we are putting these in heron/common?